config/webpack/webpack.config.js

```diff
@@ -31,8 +31,9 @@ module.exports = merge(shims, {
           options: {
             presets: [
               ['@babel/preset-env', {
-                useBuiltIns: 'usage',
-                modules: 'commonjs'
+                corejs: 2,
```

Comment:
The new version now asks for defining `corejs` and it defaults to 2, but that might change with future versions to 3.

Should be pretty trivial to update to 3 as well but that's another PR.